app/views/hiring_staff/vacancies/edit_vacancy_sections/_edit_job_summary.html.haml

```diff
@@ -0,0 +1,16 @@
+%h2.govuk-heading-m.app-task-list__section#job_summary_heading
+  %span.app-task-list__section-number 5.
+  = t('jobs.job_summary')
+  = link_to t('buttons.change'), school_job_job_summary_path(@vacancy.id), class: 'govuk-link change-button', 'aria-label': t('jobs.aria_labels.change_job_summary')
+
+%dl.app-check-your-answers.app-check-your-answers--short
+
+  .app-check-your-answers__contents
```

Comment:
In another PR @cesidio observed these are good candidates for abstracting.  I'm adding this note to forestall other change requests - my preferred path would be to start to replace some of this template code with a completely different workflow. 

This is sufficient for now-a partial or helper method would have to deal with at least two different logic branches (`.first-question`, etc.) and there are higher value things to be cleaning up right now.